File: 58-length-of-last-word/58-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        words = s.split(" ");
        max_len=0;
        for i in range(len(words)-1, -1, -1):
            if len(words[i])>0:
                leng = len(words[i])
                max_len = leng;
                break
        return max_len;

File: 58-length-of-last-word/58-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        string newstring = "";
        int i= s.length()-1;
        int count =0;
        while(s[i] == ' ')
        i--;
        while(i>=0 && s[i]!=' ')
        {
            count++;i--;
        }
        return count;
    }
};